src/libraries/token/StandardToken.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity ^0.4.11;
+
+import 'ROOT/libraries/token/BasicToken.sol';
+import 'ROOT/libraries/token/ERC20.sol';
+
+/**
+ * @title Standard ERC20 token
+ *
+ * @dev Implementation of the basic standard token.
+ * @dev https://github.com/ethereum/EIPs/issues/20
+ * @dev Based on code by FirstBlood: https://github.com/Firstbloodio/token/blob/master/smart_contract/FirstBloodToken.sol
+ */
+contract StandardToken is ERC20, BasicToken {
+	using SafeMath for uint256;
+
+	mapping (address => mapping (address => uint256)) allowed;
+
+	/**
+	* @dev Transfer tokens from one address to another
+	* @param _from address The address which you want to send tokens from
+	* @param _to address The address which you want to transfer to
+	* @param _value uint256 the amout of tokens to be transfered
+	*/
+	function transferFrom(address _from, address _to, uint256 _value) returns (bool) {
+		var _allowance = allowed[_from][msg.sender];
```

Comment:
I'm a fan of `var` personally, but if as a team we would prefer to be explicit (which so far it sounds like we are leaning that way) I will get on board with that.